renovate.json5

```diff
@@ -0,0 +1,192 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":gitSignOff",
+    "helpers:pinGitHubActionDigests"
+  ],
+  // This ensures that the gitAuthor and gitSignOff fields match
+  "gitAuthor": "renovate[bot] <bot@renovateapp.com>",
+  "includePaths": [
+    ".github/workflows/**",
+    "Dockerfile",
+    "go.mod",
+    "go.sum",
+    "Makefile",
+  ],
+  postUpdateOptions: [
+    "gomodTidy"
+  ],
+  "ignoreDeps": [
+    "github.com/cilium/cilium"
+  ],
+  "pinDigests": true,
+  "ignorePresets": [":prHourlyLimit2"],
+  "separateMajorMinor": true,
+  "separateMultipleMajor": true,
+  "separateMinorPatch": true,
+  "pruneStaleBranches": true,
+  "baseBranches": [
+    "master",
+    "v0.11"
+  ],
+  "vulnerabilityAlerts": {
+    "enabled": true
+  },
+  "labels": [
+    "kind/enhancement",
+    "release-note/misc",
+    "priority/release-blocker"
+  ],
+  "stopUpdatingLabel": "renovate/stop-updating",
```

Comment:
this tag doesn't exist yet in the repository.

Reply:
New label added

Reply:
Thanks, updated it to the same description and color as in cilium/cilium for consistency.